config_sample.yml

```diff
@@ -2,9 +2,9 @@
 #Copy its content to config.yml file
 #It is used only when you run tests using IDE
 
-browser: FF #acceptable values: FF, CHROME, GHOST, ANDROID
+browser: CHROME #acceptable values: FF, CHROME, GHOST, ANDROID
 wikiName: mediawiki119 #default testing wiki
-env: prod #acceptable values: prod, preview, sandbox-number dev-name
+env: preview #acceptable values: prod, preview, sandbox-number dev-name
 credentialsPath: c:/selenium-config/config.xml #Path to your config.xml file
```

Comment:
Changes made to config_sample should not be merged. Add this file to your .gitignore